### DISCUSSION TITLE
Linux’s new page reclaim policy

### DISCUSSION THREAD

**Discussion by rsssxd** [Divested-Mobile/DivestOS-Build#98] [Ideas] [answered]
I know DivestOS mostly supports older devices.

Is it possible to bring the following Linux feature to older Linux kernels? And to older Android versions.

https://blog.esper.io/android-dessert-bites-22-linux-memory-management-38419756/

**Comment by SkewedZeppelin** [comment, marked as answer]
Extremely unlikely.
